src/forwarder/internal/pkg/authz/write.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 Cisco and/or its affiliates.
```

Comment:
Should we use `Copyright (c) 2020 Doc.ai and/or its affiliates`?

Reply:
yes, we need to use this pattern